internal/endpoints/metrics.go

```diff
@@ -75,8 +80,12 @@ func IncMessageProcessErrors() {
 	messageProcessError.With(p.Labels{}).Inc()
 }
 
-func IncInvalidRequests() {
-	invalidRequests.With(p.Labels{}).Inc()
+func IncInvalidConsumerRequests() {
```

Comment:
Similar issue here. The name looks like it's referring to a request into the consumer is bad rather than the requestID is invalid.